alacritty/src/config/mod.rs

```diff
@@ -222,6 +222,11 @@ fn load_imports(config: &Value, config_paths: &mut Vec<PathBuf>, recursion_limit
             },
         };
 
+        if !path.exists() {
+            info!(target: LOG_TARGET_CONFIG, "Skip importing: '{}' not found", path.display());
```

Comment:
I'm not sure whether it should be filtered here, maybe we should do it earlier? I'm not sure if there's a reason to watch for config file that can appear after some period of time.

Also, I don't quite like the way this `info` message looks, maybe it should follow our message about loaded config and be multiline, I'd suggest something like that:

```rust
info!("Skipping importing config; not found:");
info!("  {:?}", path);
```

Reply:
I filtered paths here as this is right after they are parsed from config and before they are added to the config list. If we would want to support watching these files then I would move the check to after the path is added to the config_paths list and before attempting to read its file contents. 

Do we want to support watching for files that do not exist? I played with the idea in `monitor::watch()`. Bypassing the `fs::canonicalize` for testing it did work when an import file was touched and caught with the notify watcher. Apparently renames are currently not caught at least on windows (which I was testing). [165](https://github.com/notify-rs/notify/issues/165) and [166](https://github.com/notify-rs/notify/pull/166).

The check for path exist would depend on how config watchers should be handled for files that don't exist yet.

My option, it would be interesting to support but not worth the change. You can also touch/save any other alacrity config file and it will reload the config the new file will be caught. 

As for the info message I can change it to your suggestion.

Reply:
Ah, yeah, you're right, it's in the current place. So just changing a message should be fine. I forgot that `parse_config` calls `load_imports`, hence recursion.